### PR TITLE
fix: MD005/list-indent

### PR DESCRIPTION
**Pull request by nschonni**
Inconsistent indentation for list items at the same level

### CODE REVIEW

**Comment by e0i** [comment]
@AndreaBarr Please review and merge this housekeeping PR that introduces various whitespace indentation fixes. Thanks.

File: windows/client-management/mdm/policy-csp-devicelock.md [hunk 1]
```diff
@@ -387,12 +387,12 @@ Specifies whether device lock is enabled.
 > [!Important]
 > **DevicePasswordEnabled** should not be set to Enabled (0) when WMI is used to set the EAS DeviceLock policies given that it is Enabled by default in Policy CSP for back compat with Windows 8.x. If **DevicePasswordEnabled** is set to Enabled(0) then Policy CSP will return an error stating that **DevicePasswordEnabled** already exists. Windows 8.x did not support DevicePassword policy. When disabling **DevicePasswordEnabled** (1) then this should be the only policy set from the DeviceLock group of policies listed below:
 > - **DevicePasswordEnabled** is the parent policy of the following:
-> 	- AllowSimpleDevicePassword
->	- MinDevicePasswordLength
->	- AlphanumericDevicePasswordRequired
->		- MinDevicePasswordComplexCharacters
```
Comment (nschonni) [inline windows/client-management/mdm/policy-csp-devicelock.md]: Couldn't tell if this was intentionally below the previous item

File: windows/security/threat-protection/microsoft-defender-atp/user-roles.md [hunk 2]
```diff
@@ -34,31 +34,31 @@ The following steps guide you on how to create roles in Microsoft Defender Secur
...
-	 - **Role name**
-	 - **Description**
-	 - **Permissions**
-		  - **View data** - Users can view information in the portal.
-		  - **Alerts investigation** - Users can manage alerts, initiate automated investigations, collect investigation packages, manage machine tags, and export machine timeline.
-		  - **Active remediation actions** - Users can take response actions and approve or dismiss pending remediation actions.
+    - **Role name**
+    - **Description**
+    - **Permissions**
+          - **View data** - Users can view information in the portal.
```
Comment (nschonni) [inline windows/security/threat-protection/microsoft-defender-atp/user-roles.md]: Render changed here, I'll need to take a look

**Comment by illfated** [comment]
Looks like the merge of your previous PR to this file brought about the need for a branch update for this PR.

**Comment by e0i** [comment]
@AndreaBarr Following up for updates.

**Comment by illfated** [comment]
Another PR branch rebase be useful before someone gets around to merging this content.

**Comment by AndreaBarr** [comment]
@nschonni  please resolve the conflicts so that I can merge this PR

**Comment by nschonni** [comment]
@AndreaBarr I've rebased again

**Pull request merged**